src/device/mod.rs

```diff
@@ -863,7 +874,7 @@ impl RequestedState {
 }
 
 impl MeshnetEntites {
```

Comment:
Can't help but notice a typo in this struct :)

Reply:
Done